frontend/src/features/documents/documentDetail/DocumentDetailContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import { Api_Document } from 'models/api/Document';
+import { ExternalResultStatus } from 'models/api/ExternalResult';
+import { useEffect, useState } from 'react';
+
+import { ComposedDocument } from './ComposedDocument';
+import DocumentDetailView from './DocumentDetailView';
+import { useDocumentProvider } from './hooks/useDocumentProvider';
+
+export interface IDocumentDetailContainerProps {
+  pimsDocument: Api_Document;
+}
+
+export const DocumentDetailContainer: React.FunctionComponent<IDocumentDetailContainerProps> = props => {
+  const [document, setDocument] = useState<ComposedDocument>({ pimsDocument: props.pimsDocument });
+
+  const { retrieveDocumentMetadata, retrieveDocumentMetadataLoading } = useDocumentProvider();
+  useEffect(() => {
+    const fetch = async () => {
+      if (props.pimsDocument.mayanDocumentId !== undefined) {
+        const axiosResponse = await retrieveDocumentMetadata(props.pimsDocument.mayanDocumentId);
+        console.log(axiosResponse?.status, ExternalResultStatus.Success);
+        if (axiosResponse?.status === ExternalResultStatus.Success) {
+          let results = axiosResponse.payload.results;
+          setDocument(document => ({
+            ...document,
+            mayanMetadata: results,
+          }));
+        }
+      }
+    };
+
+    fetch();
+  }, [props.pimsDocument.mayanDocumentId, retrieveDocumentMetadata]);
+  return <DocumentDetailView metadata={document} isLoading={retrieveDocumentMetadataLoading} />;
```

Comment:
We need to come up with a naming standard for forms. We have been using *View and *Form - would be nice to pick one. Can discuss this Friday